perk-store-webapps/src/main/webapp/css/main.less

```diff
@@ -1421,9 +1421,15 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
 
    @media (max-width: 599px) {
-    #perkstoreOrderPortlet {
```

Comment:
It's hard to believe that this is due to Perk Store CSS, I think it's due to the new re-usable component used in Wallet Settings.